app/src/main/java/com/hedvig/app/feature/chat/viewmodel/ChatViewModel.kt

```diff
@@ -293,4 +313,17 @@ class ChatViewModel(
             gifs.postValue(response.getOrNull()?.data)
         }
     }
+
+    fun restartChat() {
```

Comment:
This is a bit gnarly, tried to keep the same logic as in /develop but still dont get a proper restart. I think we should look into this more since it feels somewhat broken both in this branch and in develop - first the app restarts and after a delay the chat starts again, but with no option to go back.

Reply:
I agree that this is probably broken, unfortunately. 😞